executor/prepared.go

```diff
@@ -244,6 +244,8 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 
 	preparedObj := &plannercore.CachedPrepareStmt{
 		PreparedAst:         prepared,
+		StmtText:            prepared.Stmt.Text(),
+		StmtDB:              e.ctx.GetSessionVars().CurrentDB,
```

Comment:
Will the following case cause the wrong result?
```
use test;
prepare stmt from `select * from plan_cache.t`;
```
The test contains the `db`, but it is not same with `e.ctx.GetSessionVars().CurrentDB`

Reply:
Maybe we can use the `CachedPrepareStmt.NormalizedSQL` directly?

Reply:
> Will the following case cause the wrong result?
> 
> ```
> use test;
> prepare stmt from `select * from plan_cache.t`;
> ```
> 
> The prepare text contains the `db`, but it is not same with `e.ctx.GetSessionVars().CurrentDB`

There is no problem about it. The `CachedPreparedStmt.StmtDB` is only used to build the plan cache key, and the DB to accessed is determined by `CachedPreparedStmt.PreparedAst`.

Reply:
> Maybe we can use the `CachedPrepareStmt.NormalizedSQL` directly?

You are right. This issue was fixed.